src/test/kotlin/org/rust/lang/core/resolve2/ResolveCommonThreadPoolImpl.kt

```diff
@@ -0,0 +1,28 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.core.resolve2
+
+import com.intellij.testFramework.ThreadTracker
+import java.util.concurrent.ExecutorService
+import java.util.concurrent.ForkJoinPool
+
+class TestResolveCommonThreadPoolImpl : ResolveCommonThreadPool() {
```

Comment:
Unfortunately, if service test implementations are located in test source sets of plugin A, plugin B depends on plugin A and you run tests in plugin B, you will get error because test code is not present in final zip of plugin A.
For example, EduTools plugin depends on Rust plugin.
So let's move it to main source set as we do with all other service test implementations